modules/shortcodes/instagram.php

```diff
@@ -132,6 +132,79 @@ function jetpack_instagram_embed_reversal( $content ) {
 	return $content;
 }
 
+/**
+ * List of whitelisted and sanitized parameters
+ * that can be used with the Instagram oEmbed endpoint.
+ *
+ * Those parameters can be provided via the Instagram URL, or via shortcode parameters.
+ *
+ * @see https://developers.facebook.com/docs/graph-api/reference/instagram-oembed#parameters
+ *
+ * @since 9.1.0
+ *
+ * @param string $url  URL of the content to be embedded.
+ * @param array  $atts Shortcode attributes.
+ *
+ * @return array $params Array of parameters to be used in Instagram query.
+ */
+function jetpack_instagram_get_whitelisted_parameters( $url, $atts = array() ) {
```

Comment:
```suggestion
function jetpack_instagram_get_allowed_parameters( $url, $atts = array() ) {
```

(see https://github.com/Automattic/jetpack/pull/17700/files#r516918779)

Reply:
Resolved via 7f974adec

Reply:
Thank you both. My old habits die hard :\